libcodechecker/analyze/log_parser.py

```diff
@@ -106,7 +106,13 @@ def dump_compiler_info(output_path, filename, data):
         f.write(json.dumps(all_data))
 
 
-def get_compiler_includes(compiler, lang, compile_opts, output_path=None,
+def load_compiler_info(filename, compiler):
+    with open(filename, 'r') as f:
+        data = json.load(f)
+    return data[compiler]
+
+
+def get_compiler_includes(parseLogOptions, compiler, lang, compile_opts,
```

Comment:
If we introduce a new argument at the first place in a function argument it can cause weird and hard to debug errors in python if not all of the function calls are updated.
Please recheck if there are no wrong function calls for the modified functions: `get_compiler_includes`, `get_compiler_target`, `parse_compile_commands_json`

Reply:
Checked and everthing looks OK.
```
libcodechecker/analyze/log_parser.py|24 col 19| # be forwarded to get_compiler_includes and get_compiler_defines so the 
libcodechecker/analyze/log_parser.py|119 col 5| def get_compiler_includes(parseLogOptions, compiler, lang, compile_opts,
libcodechecker/analyze/log_parser.py|253 col 21| get_compiler_includes(parseLogOptions, results.compiler,

libcodechecker/analyze/log_parser.py|152 col 5| def get_compiler_target(parseLogOptions, compiler):    
libcodechecker/analyze/log_parser.py|259 col 21| get_compiler_target(parseLogOptions, results.compiler)

libcodechecker/analyze/log_parser.py|177 col 5| def parse_compile_commands_json(logfile, parseLogOptions):
libcodechecker/analyze/log_parser.py|181 col 16| LOG.debug('parse_compile_commands_json: ' + str(add_compiler_defaults))
libcodechecker/analyze/log_parser.py|294 col 23| actions = parse_compile_commands_json(logfile, parseLogOptions)
tests/unit/test_buildcmd_escaping.py|86 col 31| return log_parser.parse_compile_commands_json(text, ParseLogOptions())                    

```